SpatialGDK/Source/SpatialGDK/Public/Schema/Ping.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/Component.h"
+#include "SpatialConstants.h"
+#include "Utils/SchemaUtils.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+#include <WorkerSDK/improbable/c_worker.h>
+
+namespace improbable
+{
+
+	struct ServerPing : Component
+	{
+		static const Worker_ComponentId ComponentId = SpatialConstants::SERVER_PING_COMPONENT_ID;
+
+		ServerPing() = default;
+		ServerPing(const Worker_ComponentData& Data)
```

Comment:
If you define a copy constructor then you should also define all the other special member functions.
However generally if you have a struct with only defaults then I wouldn't bother defining any of them. They will all be correctly defaulted.

Reply:
This goes for the ClientPong too.

Reply:
Ah this isn't a copy constructor; I'm just misreading it.